other/updating-header.md

```diff
@@ -0,0 +1,60 @@
+# Changing NYPL Header
+
+The primary NYPL header is maintained as a React component distributed as an NPM module. To support non-React apps, a "header app" ensures the component may also be included via  aa `<script>` or as fully rendered HTML.
+
+The following proposes a safe sequence of deployments for activing changes to the header across our apps. 
+
+## 1. Update React Header Component
+
+The React Component [source is maintained in Github](https://github.com/NYPL/dgx-header-component) and [published to NPMJS](https://www.npmjs.com/package/@nypl/dgx-header-component).
+
+Changes to the component should follow [the instructions in the repo](https://github.com/NYPL/dgx-header-component#contributing-is-fun-and-easy) for review, merging, and tagging. Don't overlook publishing the new version to NPMJS.
+
+## 2. Activate Changes in React Apps
+
+> Note that the specific feature being rolled out may require modifying the order of deployments below. For example, if the header functionality is the creation of a cookie during login (via Encore), you may want to deploy the Header App before the React apps below.
+
+Starting with the lowest traffic [React apps](https://github.com/NYPL/engineering-general/tree/master/other#reactnode) (excluding "Header App"), update the `dgx-header-component` dependency to use the new version. Proposed order:
+
+ * [Get a library card](https://github.com/NYPL/nypl-library-card-app)
+ * [New Arrivals](https://github.com/NYPL/dgx-new-arrivals)
+ * [Staff picks](https://github.com/NYPL/staff-picks)
+ * [BookLists](https://github.com/NYPL/dgx-booklists)
+ * [SCC](https://github.com/NYPL-discovery/discovery-front-end)
+ * [Global Search](https://github.com/NYPL/dgx-global-search)
+ * [Blogs beta](https://github.com/NYPL/dgx-blogs)
+ * [Homepage](https://github.com/NYPL/dgx-homepage)
+
+For each of these:
```

Comment:
I would _LOVE_ to look into doing these steps, in the downwind repos using Travis or (maybe better?) GitHub actions.

Reply:
I would love to hear about automating these steps more. They're all already push-to-deploy via Travis, but I'd be interested in orchestrating them as a unit. I know nothing of actions github..